portal/static/js/psaTracker.js

```diff
@@ -631,15 +660,25 @@
                     .attr("class", "axis-stroke")
                     .style("text-anchor", "end");
 
+                //borders around graph area
+                var BORDER_STROKE_COLOR = "#3F4B54";
+                svg.append("path").attr("d", "M " + margin.left + " " + margin.top + " V " + (HEIGHT-margin.bottom) + " H" + (WIDTH-margin.right))
```

Comment:
If this is modifying an existing static SVG (.svg file), could you make the modifications to the file directly?

Reply:
@ivan-c ivan, actually this is drawing the borders of the PSA data graph as it should appear on the page per prototype - so not modifying any static SVG.